cclient/response/get_neighbors.c

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include "get_neighbors.h"
+
+void neighbor_info_t_copy(void* _dst, const void* _src) {
+  neighbor_info_t *dst = (neighbor_info_t*)_dst, *src = (neighbor_info_t*)_src;
+  dst->address = src->address;
+  dst->all_trans_num = src->all_trans_num;
+  dst->invalid_trans_num = src->invalid_trans_num;
+  dst->new_trans_num = src->new_trans_num;
+}
+
+void neighbor_info_t_dtor(void* _elt) {
+  neighbor_info_t* elt = (neighbor_info_t*)_elt;
+  if (elt->address) {
+    char_buffer_free(elt->address);
+  }
+}
+
+UT_icd ut_neighbor_icd = {sizeof(neighbor_info_t), NULL, neighbor_info_t_copy,
+                          neighbor_info_t_dtor};
+
+get_neighbors_res_t* get_neighbors_res_new() {
+  get_neighbors_res_t* nbors = malloc(sizeof(get_neighbors_res_t));
```

Comment:
Can you cast the result of `malloc` to `get_neighbors_res_t*` to avoid warnings if compiled in C++ ?

Reply:
sure